deepmd/pt/model/model/dp_atomic_model.py

```diff
@@ -100,6 +101,10 @@ def get_sel(self) -> List[int]:
         """Get the neighbor selection."""
         return self.sel
 
+    def get_nnei(self) -> List[int]:
```

Comment:
Perhaps [`torch.jit.export`](https://pytorch.org/docs/stable/jit.html#torch.jit.export) should be used?